adapters/base_embeddings.py

```diff
@@ -0,0 +1,80 @@
+import openai
+from openai import NOT_GIVEN
+import dtlpy as dl
+import os
+import logging
+
+logger = logging.getLogger("BaseEmbeddingsModelAdapter")
+
+
+class ModelAdapter(dl.BaseModelAdapter):
+    def load(self, local_path, **kwargs):
+        """Load configuration for Databricks adapter"""
+        api_key = os.environ.get("DATABRICKS_API_KEY")
+        base_url = self.configuration.get("base_url")
+    
+        if not api_key:
+            raise ValueError(
+                "Environment variable 'DATABRICKS_API_KEY' is not set. "
+                "Please set it to proceed."
+            )
```

Comment:
the secret name can be any name, not necessarily 'DATABRICKS_API_KEY'

Reply:
@RoniDataloopAi the previous dbrx code i.e., the base_chat_completion.py does it this way -- do you have a recommended alternative?